tensilelite/Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -355,11 +365,18 @@ namespace Tensile
         uint32_t wgmRemainder1            = 0;
         uint32_t magicNumberWgmRemainder1 = 0;
 
+        if((sizeMapping.globalAccumulation == 2) || (sizeMapping.customKernelName != ""))
+        {
+            args.template append<uint32_t>("GSUSync", 0);
```

Comment:
Don't put internal args in the middle of the user args
The format should be
Gemm:
user args + internal args 
Grouped gemm:
Wgtable + user args + internal args